packages/keystone/src/lib/core/mutations/nested-mutation-many-input-resolvers.ts

```diff
@@ -24,7 +25,7 @@ async function getDisconnects(
       uniqueWheres.map(async filter => {
         if (filter === null) return [];
         try {
-          await context.sudo().db.lists[foreignList.listKey].findOne({ where: filter });
+          await findOne({ where: filter }, foreignList, context);
```

Comment:
This isn't going through GraphQL here so when `resolveUniqueWhereInput` throws an error, it won't be wrapped.

(and this is also more correct because the values won't go through GraphQL coercion again)